README.md

```diff
@@ -1,16 +1,19 @@
 Developer guide to Tessel 2 Docs Gitbook
```

Comment:
That's awesome! That will make it a bit more approachable for editorial folks who would like to contribute. :smile: